doc/source/ray-core/doc_code/anti_pattern_return_ray_put.py

```diff
@@ -0,0 +1,93 @@
+import ray
+import numpy as np
+
+
+@ray.remote
```

Comment:
The code example is really long and has several different parts. Small values vs large values, single values vs multiple, tasks vs actors. Can we break this up into the different examples and put the relevant text explaining it before each code block?